### DISCUSSION TITLE
Dropdown with checkboxes closes after each selection

### DISCUSSION THREAD

**Discussion by pablo-gonzalez-helpwan** [livewire/flux#143] [Feature Requests]
I'm not sure if it's a bug, but it does feel annoying.

I have a dropdown with many checkboxes, and after each selection (clicking in a checkbox), the dropdown closes. That could be fine for radio, but for checkboxes I'd like the option to keep it open to toggle many before closing the dropdown.

It can be reproduced with the example code in the docs:

```
<flux:dropdown>
    <flux:button icon-trailing="chevron-down">Permissions</flux:button>

    <flux:menu>
        <flux:menu.checkbox wire:model="read" checked>Read</flux:menu.checkbox>
        <flux:menu.checkbox wire:model="write" checked>Write</flux:menu.checkbox>
        <flux:menu.checkbox wire:model="delete">Delete</flux:menu.checkbox>
    </flux:menu>
</flux:dropdown>
```

**Comment by joshhanley** [comment]
Thanks for reporting! We have converted this issue to a feature request for now, so we can focus on fixing any bugs first.

As there is already a discussion #130 open for this, I'm going to close this discussion, so we can continue the conversation there.
